client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -726,6 +727,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 					globalStylesInPersonalPlan={ globalStylesInPersonalPlan }
 					siteId={ site.ID }
 					stylesheet={ selectedDesign.recipe?.stylesheet }
+					screenshot={ selectedDesign.screenshot }
```

Comment:
Nit: Is there an easy way to pass the image set that is generated with photon here?

https://github.com/Automattic/wp-calypso/blob/20e370b485f6032d5e83148d9f86750caf2e4252/packages/design-picker/src/components/unified-design-picker.tsx#L40-L52

On higher-resolution screens (<1440px), the text on the screenshot starts to look a bit blurry.
![CleanShot 2023-08-22 at 16 30 06@2x](https://github.com/Automattic/wp-calypso/assets/11555574/e3d655b8-917a-4752-966b-bd5a5d36b7a1)

Reply:
I don't think there is a easy way as this logic is in a *separate* package and we cannot reference it in Calypso, but we already have this logic in Calypso and we should be able to centralize it in on place to use.

---

That said, I think screenshot pictures are usually wide by default, and as you can see in Makoney case the issue is that the original file is blurred. [screenshot file](https://theme.files.wordpress.com/2023/01/makoney-preview-2.jpg)

Reply:
Is there any way we can re-use the same image we display in the theme showcase? That one seems much sharper:

Theme showcase | Design picker
--- | ---
<img width="1483" alt="Screenshot 2023-08-23 at 10 45 25" src="https://github.com/Automattic/wp-calypso/assets/1233880/2d89a34c-885f-4f76-be30-a7aa56511f80"> | <img width="1959" alt="Screenshot 2023-08-23 at 10 45 44" src="https://github.com/Automattic/wp-calypso/assets/1233880/5bd8adb7-fcdf-4f28-9673-c1f2a8dd5afd">
[Source](https://i0.wp.com/theme.files.wordpress.com/2022/12/makoney-wordpress-block-theme-for-full-site-editing.png?ssl=1&w=735&zoom=2) | [Source](https://theme.files.wordpress.com/2023/01/makoney-preview-2.jpg)

Reply:
Fixed on https://github.com/Automattic/wp-calypso/pull/80927/commits/d7cde849c9d824493e37b5e2447ae791e4912975, now the full page screenshot is displayed.